diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -298,6 +314,27 @@ namespace diff_drive_controller
 
   void DiffDriveController::update(const ros::Time& time, const ros::Duration& period)
   {
+    // UPDATE DYNAMIC PARAMS
+    // Retreive dynamic params:
+    DynamicParams dynamic_params = *(dynamic_params_.readFromRT());
```

Comment:
The problem is that there's not write from RT methods supported: see [this proposal](https://github.com/ros-controls/realtime_tools/issues/14).

I could add a comment with that link, and we can go back to this discussion when this goes upstream; this is fast enough anyway. What do you think?